### PR TITLE
API-924: Add max length to reference and PO number

### PR DESCRIPTION
**Pull request by ashwini-cimpress**

### CODE REVIEW

File: docs/Changelog.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Changelog
 ==========
 
 
+05-08-2020 ``v2``
+    We set max lenght limit for `reference` and `poNumber` fields to 35 in create order API.
```
Comment (RogierDonkervliet) [inline docs/Changelog.rst:8]: Typo in length

Comment (RogierDonkervliet) [reply docs/Changelog.rst:8]: Maybe you could be a bit more informative on what 35 is? Is the limit 35 beers? :P I'd suggest `35 characters`

File: docs/Changelog.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ Changelog
 
 
 05-08-2020 ``v2``
-    We set max lenght limit for `reference` and `poNumber` fields to 35 in create order API.
+    We set max length limit for `reference` and `poNumber` fields to 35 in create order API.
```
Comment (RogierDonkervliet) [inline docs/Changelog.rst:8]:
```suggestion
    We set max length limit for `reference` and `poNumber` fields to 35 characters in create order API.
```

File: docs/Changelog.rst (hunk 1 above)
Comment (Avariya) [reply docs/Changelog.rst]: 25 kratten and kassbrooden

**Pull request merged**